egs/wsj/s5/steps/nnet3/compute_and_adjust_priors.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+
+# Copyright 2016    Pegah Ghahremani.
+# Apache 2.0.
+
+""" This script is based on steps/nnet3/tdnn/train.sh
+"""
+
+import argparse
+import logging
+import os
+import pprint
+import shutil
+import sys
+import traceback
+
+sys.path.insert(0, 'steps')
+import libs.nnet3.train.common as common_train_lib
+import libs.common as common_lib
+import libs.nnet3.train.frame_level_objf as train_lib
+import libs.nnet3.report.log_parse as nnet3_log_parse
+
+
+logger = logging.getLogger('libs')
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler()
+handler.setLevel(logging.INFO)
+formatter = logging.Formatter("%(asctime)s [%(filename)s:%(lineno)s - "
+                              "%(funcName)s - %(levelname)s ] %(message)s")
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+logger.info('Starting compute and adjusting posteriors (train_dnn.py)')
```

Comment:
I mean change 'train_dnn.py' in the log statement to the current script name 'compute_and_adjust_priors.py'